Tests/mock_server.py

```diff
@@ -527,6 +554,9 @@ def run_with_mock(proxy_instance: MITMProxy, playbook_or_integration_id: str, re
                 proxy_instance.move_mock_file_to_repo(playbook_or_integration_id)
                 proxy_instance.successful_rerecord_count += 1
                 proxy_instance.rerecorded_tests.append(playbook_or_integration_id)
+                if proxy_instance.should_update:
+                    proxy_instance.logging_module.debug("committing new/updated mock files to mock git repo.")
```

Comment:
should we also log here `playbook_or_integration_id` or it will be clear from previous logs?

Reply:
It will be clear IMO